src/pages/AssetMapPage/AMDrawer.tsx

```diff
@@ -89,6 +119,35 @@ export const AMDrawer = () => {
   const rawProfiles = useMapResults();
   const { measures } = useMapMeasures(metric);
   const showHiddenFeatures = useDevMode();
+  const [metric2, setMetric2] = useStateAmMetric();
+  const amEpochs = useMapEpochs();
+  const [amEpochId, setAmEpochId] = useStateAmEpochId();
+
+  // This is the AssetMapPage Controller
+  useEffect(() => {
+    if (amEpochs.length === 0) {
+      return;
+    }
+    setAmEpochId(amEpochs[amEpochs.length - 1]?.id);
+    // TODO: Load gifts for selected epoch when needed:
+    // https://linear.app/yearn/issue/APE-192/api-dont-always-load-everything-and-improve-fetching
```

Comment:
we can remove these comments now

Reply:
Removed with https://github.com/coordinape/coordinape/pull/914/commits/0fd0cee5b2f40d800b1a6cea019b23f099352955